### ISSUE TITLE
Incomplete information installing via PyPI

### ISSUE THREAD

**Issue by johnblommers** [vkbo/novelWriter#123]
So I installed novelWriter in a fairly new Linux guest using Virtualbox follows:

```bash
python3 -m pip install novelwriter --user
```

and that succeeded. However a naive user will need some guidance and first launch of novelWriter might be the time to offer the following information:

- Where to find a sample project. Consider shipping it along with the code and including a hint to its location in the File menu
- Mention why `sudo apt install pandoc` is important
- Recommend which spelling package to use. On first launch, novelWriter reports `WARNING   No spell check tool available`. Consider recommending how to install one.

**Comment by vkbo** [comment]
Although I'm pushing novelWriter to pypi, this is still undocumented exactly because there are unresolved issues. A major issue, not only for pypi, but for Windows in general, is how to handle spell checking.

The main spell check tool is spell enchant, accessed through pyenchant. However, pyenchant is no longer being maintained by its author, and is somewhat out of date.

The spell check toolbox in novelWriter is written using an internal API, so I can easily extend it to use other tools without having to modify the document editor. I'm actively looking for alternatives, but haven't found any good fits yet.

**Comment by johnblommers** [comment]
Copy that. I'm getting ahead of myself.

**Comment by vkbo** [comment]
I'm going to add hunspell support soon, as well as improve the spell checking tools.

As for access to the sample project, I will also add a splash/welcome dialog when opening novelWriter for managing projects. This will replace New project, Open project and Recent Projects in the menu. Opening the sample project will be added to the same dialog.

**Comment by vkbo** [comment]
It seems that someone has taken over the maintenance of the `pyenchant` package, and it now works on Windows 10 last I tested. For now, I have decided not to spend any more time on implementing other spell checking options.

In addition, from next minor version, pandoc will no longer be needed, so that point is no longer relevant either.

I'm yet not sure how to bundle the sample project. It should probably be shipped under the `nw/assets` folder, and copied to the user's data folder if the user requests it.
